850.split-ambiguities/d.yaml

```diff
@@ -126,6 +126,10 @@
 
 - { name: disktype, wwwpart: kamwoods, setname: disktype-kamwoods } # fork
 
+- { name: distribution, wwwpart: [philovivero, time-less-ness, wizzat], setname: "python:distribution" }
```

Comment:
Don't merge into `python:distribuition`

- That project may only contain corresponding pypi project (which is unrelated), use e.g. `distribution-tui-histograms`
- It's an application and not a python module, and these are usually merged into unprefixed name as it's how they are commonly packaged